examples/platform/mbed/capsense/capsense.cpp

```diff
@@ -0,0 +1,220 @@
+
+#include "capsense.h"
+
+using namespace std::literals::chrono_literals;
+
+#define SLIDER_NUM_TOUCH (1u)
+#define LED_OFF (1u)
+#define LED_ON (0u)
+
+#define CAPSENSE_SCAN_PERIOD_MS (20ms)
+#define EZI2C_INTERRUPT_PRIORITY (3u)
+
+#define TUNER_ENABLE (1u)
+
+const cy_stc_sysint_t CapSense_ISR_cfg = { .intrSrc = CYBSP_CSD_IRQ, .intrPriority = 4u };
+
+Semaphore capsense_sem;
+EventQueue queue;
+
+cy_stc_scb_ezi2c_context_t EZI2C_context;
+uint32_t prevBtn0Status                       = 0u;
+uint32_t prevBtn1Status                       = 0u;
+uint32_t prevSliderPos                        = 0u;
+volatile uint32_t irr                         = 0;
+volatile uint32_t fails                       = 0;
+cy_stc_syspm_callback_params_t callbackParams = { .base = CYBSP_CSD_HW, .context = &cy_capsense_context };
+
+cy_stc_syspm_callback_t capsenseDeepSleepCb = { Cy_CapSense_DeepSleepCallback,
+                                                CY_SYSPM_DEEPSLEEP,
+                                                (CY_SYSPM_SKIP_CHECK_FAIL | CY_SYSPM_SKIP_BEFORE_TRANSITION |
+                                                 CY_SYSPM_SKIP_AFTER_TRANSITION),
+                                                &callbackParams,
+                                                NULL,
+                                                NULL };
+
+cy_stc_scb_ezi2c_context_t ezi2c_context;
+cyhal_ezi2c_t sEzI2C;
+cyhal_ezi2c_slave_cfg_t sEzI2C_sub_cfg;
+cyhal_ezi2c_cfg_t sEzI2C_cfg;
```

Comment:
Do we need global variables here? Are they referenced anywhere outside this file/scope?
Consider adding 'static' to each of them or put them into unnamed namespace.

Reply:
Done